core/client/db/src/cache/mod.rs

```diff
@@ -134,21 +177,56 @@ impl<'a, Block: BlockT> DbCacheTransaction<'a, Block> {
 		mut self,
 		parent: ComplexBlockId<Block>,
 		block: ComplexBlockId<Block>,
-		authorities_at: Option<Vec<AuthorityIdFor<Block>>>,
+		data_at: HashMap<Vec<u8>, Vec<u8>>,
 		is_final: bool,
 	) -> ClientResult<Self> {
-		assert!(self.authorities_at_op.is_none());
+		assert!(self.cache_at_op.is_empty());
 
-		self.authorities_at_op = self.cache.authorities_at.on_block_insert(
-			&mut self::list_storage::DbStorageTransaction::new(
-				self.cache.authorities_at.storage(),
-				&mut self.tx
-			),
-			parent,
-			block,
-			authorities_at,
-			is_final,
-		)?;
+		// prepare list of caches that are not up-to-date
```

Comment:
```suggestion
		// prepare list of caches that are not updated
```

Reply:
Missed that comment, sorry :/ "not updated" is correct here ("not up-to-date" means opposite, which is wrong).

Reply:
Fixed.